services/src/main/java/org/apache/unomi/services/impl/segments/SegmentServiceImpl.java

```diff
@@ -1271,6 +1260,25 @@ public void run() {
         schedulerService.getScheduleExecutorService().scheduleAtFixedRate(task, initialDelay,  TimeUnit.DAYS.toSeconds(1), TimeUnit.SECONDS);
     }
 
+    private void loadScripts() throws IOException {
+        Enumeration<URL> scriptsURL = bundleContext.getBundle().findEntries("META-INF/painless", "*.painless", true);
```

Comment:
Maybe put them in the /cxs subdirectory for consistency ?

Reply:
yes, thank you for the hint, I moved them to /cxs

Reply:
Thanks you. Looks good to me now.